integration/key/release_name.go

```diff
@@ -5,3 +5,15 @@ package key
 func AppOperatorReleaseName() string {
 	return "app-operator"
 }
+
+func CustomResourceReleaseName() string {
```

Comment:
I don't see the need for extracting that to the key package. That can be arbitrary value in all tests. This won't for example if one of the tests will install multiple App CRs.

Same applies for the rest of the values. This can be declared as consts in the tests.

The idea of the `key` package is to put knowledge shared between resources (or tests in this case) in a common place. Name of created CR isn't shared knowledge IMO.

Reply:
I agree I will move these variables to `const` section.